frontend/src/components/filter/trainingReportFilters.js

```diff
@@ -0,0 +1,98 @@
+/* eslint-disable import/prefer-default-export */

```

Comment:
this looks identical to the activityReportFilters stuff. Can we just change that file to reportFilters and reuse this stuff rather than have a bunch of duplicated code?

Reply:
Lets create a ticket for this as we have been down this road before with mixed results.

Reply:
ok, did you make a ticket?